content/docs/tasks/telemetry/distributed-tracing/jaeger/index.md

```diff
@@ -32,6 +35,15 @@ To learn how Istio handles tracing, visit this task's [overview](../overview/).
 1.  When the Bookinfo application is up and running, access `http://$GATEWAY_URL/productpage` one or more times
     to generate trace information.
 
+    The number of requests you need to send in order to see trace data depends on the sampling rate set at install.
```

Comment:
at install -> at the time Istio was installed.

Reply:
Done.